model/metrics/system-metrics.yaml

```diff
@@ -0,0 +1,480 @@
+groups:
+  # General system attributes
+  - id: attributes.system
+    prefix: system
+    type: attribute_group
+    brief: "Describes System metric attributes"
+    attributes:
+      - id: device
+        type: string
+        brief: "The device identifier"
+        examples: ["(identifier)"]
+
+  # sytem.cpu.* metrics and attribute group
+  - id: attributes.system.cpu
+    prefix: system.cpu
+    type: attribute_group
+    brief: "Describes System CPU metric attributes"
+    attributes:
+      - id: state
+        type:
+          allow_custom_values: true
+          members:
+            - id: idle
+              value: 'idle'
+            - id: user
+              value: 'user'
+            - id: system
+              value: 'system'
+            - id: interrupt
+              value: 'interrupt'
+        brief: "The state of the CPU"
+        examples: ["idle", "interrupt"]
+      - id: cpu
+        type: int
+        brief: "The logical CPU number [0..n-1]"
+        examples: [1]
+
+  - id: metric.system.cpu.time
+    type: metric
+    metric_name: system.cpu.time
+    brief: "Seconds each logical CPU spent on each mode"
+    instrument: counter
+    unit: "s"
+    attributes:
+      - ref: system.cpu.state
+      - ref: system.cpu.cpu
+
+  - id: metric.system.cpu.utilization
+    type: metric
+    metric_name: system.cpu.utilization
+    brief: "Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of logical CPUs"
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: system.cpu.state
+      - ref: system.cpu.cpu
+
+  - id: metric.system.cpu.physical.count
+    type: metric
+    metric_name: system.cpu.physical.count
+    brief: "Reports the number of actual physical processor cores on the hardware"
+    instrument: updowncounter
+    unit: "{cpu}"
+    attributes: []
+
+  - id: metric.system.cpu.logical.count
+    type: metric
+    metric_name: system.cpu.logical.count
+    brief: "Reports the number of logical (virtual) processor cores created by the operating system to manage multitasking"
+    instrument: updowncounter
+    unit: "{cpu}"
+    attributes: []
+
+  # sytem.memory.* metrics and attribute group
+  - id: attributes.system.memory
+    prefix: system.memory
+    type: attribute_group
+    brief: "Describes System Memory metric attributes"
+    attributes:
+      - id: state
+        type:
+          allow_custom_values: true
+          members:
+            - id: used
+              value: 'used'
+            - id: free
+              value: 'free'
+            - id: cached
+              value: 'cached'
```

Comment:
Similar to the cpu state comment. Since we are listing `cached` here, shouldn't we then also list `buffers` and `shared` here?

And what about `total`? Is it always the case that `total` = `used` + `free` ?

Reply:
Good point, I think adding those make sense, they were just missing. Added in [4f88215](https://github.com/open-telemetry/semantic-conventions/pull/89/commits/4f882159f7967f9c5c33bcf08d84e4118a4c3c1f)